### ISSUE TITLE
hosts added via API disappear after icinga restart

### ISSUE THREAD

**Issue by artem-kosenko** [Icinga/icinga2#6580]
I faced the issue on v2.6.0 and v2.9.0 icinga. also I've found the related bug, looks like it is already closed, but not fixed.
https://github.com/Icinga/icinga2/issues/4082
https://github.com/Icinga/icinga2/issues/3668

to reproduce it:
- create host via API request
- restart icinga from console
- delete hosts via API
- create hosts via API <-- on this step I can get info about this host via API, but it is absent on the web-interface. 

here my files.
```
/var/lib/icinga2/api/packages/_api/
├── active.conf
├── active-stage
├── include.conf
└── host-name.example.com-1535636549-1
    ├── conf.d
    │   ├── downtimes
    │   └── hosts
    │       └── test-host.example.conf
    ├── include.conf
    └── zones.d

# cat /var/lib/icinga2/api/packages/_api/active.conf
if (!globals.contains("ActiveStages")) {
  globals.ActiveStages = {}
}

if (globals.contains("ActiveStageOverride")) {
  var arr = ActiveStageOverride.split(":")
  if (arr[0] == "_api") {
    if (arr.len() < 2) {
      log(LogCritical, "Config", "Invalid value for ActiveStageOverride")
    } else {
      ActiveStages["_api"] = arr[1]
    }
  }
}

if (!ActiveStages.contains("_api")) {
  ActiveStages["_api"] = "host-name.example.com-1535636549-1"
}

# cat /var/lib/icinga2/api/packages/_api/active-stage
host-name.example.com-1535636549-1

# cat /var/lib/icinga2/api/packages/_api/include.conf
include "*/include.conf"

# cat /var/lib/icinga2/api/packages/_api/host-name.example.com-1535636549-1/conf.d/hosts/test-host.example.com.conf 
object Host "test-host.example.com" {
	import "P2-host"

	address = "test-host.example.com"
	display_name = "test-host.example.com"
	notes = "my notes"
	notes_url = "http://test-host.example.com"
	vars["args"] = {
		services = {
			check_snmp_mem = {
				arg1 = "someone"
				arg2 = "90,0"
				arg3 = "100,30"
				name = "MEMORY"
			}
			ftp = {
				arg1 = 20.000000
				arg2 = 10.000000
				name = "FTP"
			}
		}
	}
	vars["facts"] = {
		nrpe = [ "check_disk", "check_file_exist" ]
		services = [ "ssh", "ftp" ]
		services_p3 = [ "load", "check_snmp_mem" ]
	}
	version = 1535637140.067982
	zone = "some-zone"
}

# cat /var/lib/icinga2/api/packages/_api/host-name.example.com-1535636549-1/include.conf 
include "../active.conf"
if (ActiveStages["_api"] == "host-name.example.com-1535636549-1") {
  include_recursive "conf.d"
  include_zones "_api", "zones.d"
}
```

**Comment by artem-kosenko** [comment]
one more think. the folder host-name.example.com-1535636549-1 is always the same. it does not change if case of any changes (remove/add new hosts.). And "active-stage" always content its name.

**Comment by artem-kosenko** [comment]
I found some workaround. It works pretty nice for me
```find /var/lib/icinga2/api/packages/_api/host-name.example.com-*/conf.d/hosts/ -name '*.conf' -mmin -1 | grep -q mg-mon.thcore.pamgservices.net && systemctl reload icinga2```

**Comment by artem-kosenko** [comment]
workaround that I use for now. It works pretty nice and resolve the issue.
``` in crontab:

* * * * * find /var/lib/icinga2/api/packages/_api/host-name.example.com-*/conf.d/hosts/ -name '*.conf' -mmin -1 | grep -q mg-mon.thcore.pamgservices.net && systemctl reload icinga```

**Comment by dnsmichi** [comment]
That's been fixed for 2.11, snapshot packages are available for testing.
